public/locales/en/status.json

```diff
@@ -45,7 +45,7 @@
     "no-button": "No",
     "description": "Are you sure you want to go back?"
   },
-  "no-match-title": "Other ways to get your status.",
+  "other-ways-to-status": "Other ways to get your status.",
```

Comment:
You are still using `no-match-title` so don't delete it. The match title might need to be just "Contact us" as they already got their status, they  don't need to get it again.

Reply:
you're right!